src/main/java/it/eng/idsa/businesslogic/processor/consumer/websocket/server/FileRecreatorBeanServer.java

```diff
@@ -17,103 +17,113 @@
 import it.eng.idsa.businesslogic.configuration.WebSocketServerConfiguration;
 import it.eng.idsa.businesslogic.service.impl.RejectionMessageServiceImpl;
 import it.eng.idsa.businesslogic.util.RejectionMessageType;
+import org.springframework.beans.factory.annotation.Value;
 
 /**
- * 
  * @author Milan Karajovic and Gabriele De Luca
- *
  */
 
 public class FileRecreatorBeanServer implements Runnable {
```

Comment:
- The commits in this class is not clear. Please, could you tell us what did you changed and added in this class?